soc/egis/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_ANDES_V5
```

Comment:
no hijacking of Kconfigs, this Kconfig has nothing to do with this soc, remove instances of it below